<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Showing information about a package — packageDetails • packagefinder</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
<!-- Bootstrap -->
<link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous" />


<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script>

<!-- bootstrap-toc -->
<link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Showing information about a package — packageDetails" />
<meta property="og:description" content="Shows detailed CRAN information for a package." />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body data-spy="scroll" data-target="#toc">
    <div class="container template-reference-topic">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">packagefinder</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">0.3.3</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jsugarelli/packagefinder/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Showing information about a package</h1>
    <small class="dont-index">Source: <a href='https://github.com/jsugarelli/packagefinder/blob/master/R/packagefinder.r'><code>R/packagefinder.r</code></a></small>
    <div class="hidden name"><code>packageDetails.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Shows detailed CRAN information for a package.</p>
    </div>

    <pre class="usage"><span class='fu'>packageDetails</span><span class='op'>(</span><span class='va'>package</span>, brief <span class='op'>=</span> <span class='cn'>FALSE</span>, show.tip <span class='op'>=</span> <span class='cn'>TRUE</span>, index <span class='op'>=</span> <span class='cn'>NULL</span><span class='op'>)</span></pre>

    <h2 class="hasAnchor" id="arguments"><a class="anchor" href="#arguments"></a>Arguments</h2>
    <table class="ref-arguments">
    <colgroup><col class="name" /><col class="desc" /></colgroup>
    <tr>
      <th>package</th>
      <td><p>Either the name of a package (capitalization does generally not matter) or the search result number shown in the results of <code><a href='findPackage.html'>findPackage</a>()</code> (the number in the <code>GO</code> column). Only one package is allowed here.</p></td>
    </tr>
    <tr>
      <th>brief</th>
      <td><p>If <code>"TRUE"</code>, only title, short and long description as well as the maintainer of the package are shown, otherwise all available fields are displayed.</p></td>
    </tr>
    <tr>
      <th>show.tip</th>
      <td><p>If <code>"TRUE"</code>, tips for getting additional information on the package are shown.</p></td>
    </tr>
    <tr>
      <th>index</th>
      <td><p>Either a path (or URL) to a search index, or a search index that is already loaded. If no index is provided, <code>packageDetails()</code> creates an ad hoc search index.</p></td>
    </tr>
    </table>

    <h2 class="hasAnchor" id="value"><a class="anchor" href="#value"></a>Value</h2>

    <p>No return value.</p>
    <h2 class="hasAnchor" id="author"><a class="anchor" href="#author"></a>Author</h2>

    <p>Joachim Zuckarelli <a href='mailto:joachim@zuckarelli.de'>joachim@zuckarelli.de</a></p>

    <h2 class="hasAnchor" id="examples"><a class="anchor" href="#examples"></a>Examples</h2>
    <pre class="examples"><div class='input'> <span class='fu'>packageDetails</span><span class='op'>(</span><span class='st'>"ggplot2"</span><span class='op'>)</span> 
</div><div class='output co'>#&gt; <span style='color: #00BBBB;'>
#&gt; Package </span><span style='color: #00BBBB;font-weight: bold;'>ggplot2</span><span style='color: #00BBBB;'> 
#&gt; 
#&gt; </span><span style='color: #555555;'>Title: </span><span>Create Elegant Data Visualisations Using the Grammar of Graphics
#&gt; 
#&gt; </span><span style='color: #555555;'>Long description: </span><span>
#&gt; A system for 'declaratively' creating graphics,
#&gt;     based on "The Grammar of Graphics". You provide the data, tell 'ggplot2'
#&gt;     how to map variables to aesthetics, what graphical primitives to use,
#&gt;     and it takes care of the details.
#&gt; 
#&gt; </span><span style='color: #555555;'>Publish date: </span><span>2020-12-30
#&gt; 
#&gt; </span><span style='color: #555555;'>Version: </span><span>3.3.3
#&gt; 
#&gt; </span><span style='color: #555555;'>License: </span><span>MIT + file LICENSE
#&gt; 
#&gt; </span><span style='color: #555555;'>Maintainer: </span><span>Thomas Lin Pedersen &lt;thomas.pedersen@rstudio.com&gt;
#&gt; 
#&gt; </span><span style='color: #555555;'>URL: </span><span>https://ggplot2.tidyverse.org,
#&gt; https://github.com/tidyverse/ggplot2
#&gt; 
#&gt; </span><span style='color: #555555;'>Bug reports: </span><span>https://github.com/tidyverse/ggplot2/issues
#&gt; 
#&gt; </span><span style='color: #555555;'>Authors: </span><span>
#&gt; Hadley Wickham [aut] (&lt;https://orcid.org/0000-0003-4757-117X&gt;),
#&gt;   Winston Chang [aut] (&lt;https://orcid.org/0000-0002-1576-2126&gt;),
#&gt;   Lionel Henry [aut],
#&gt;   Thomas Lin Pedersen [aut, cre]
#&gt;     (&lt;https://orcid.org/0000-0002-5147-4711&gt;),
#&gt;   Kohske Takahashi [aut],
#&gt;   Claus Wilke [aut] (&lt;https://orcid.org/0000-0002-7470-9261&gt;),
#&gt;   Kara Woo [aut] (&lt;https://orcid.org/0000-0002-5125-4188&gt;),
#&gt;   Hiroaki Yutani [aut] (&lt;https://orcid.org/0000-0002-3385-7233&gt;),
#&gt;   Dewey Dunnington [aut] (&lt;https://orcid.org/0000-0002-9415-4582&gt;),
#&gt;   RStudio [cph, fnd]
#&gt; 
#&gt; </span><span style='color: #555555;'>Authors@R: </span><span>
#&gt; c(
#&gt;     person("Hadley", "Wickham", , "hadley@rstudio.com", "aut",
#&gt;            comment = c(ORCID = "0000-0003-4757-117X")),
#&gt;     person("Winston", "Chang", , role = "aut",
#&gt;            comment = c(ORCID = "0000-0002-1576-2126")),
#&gt;     person("Lionel", "Henry", , role = "aut"),
#&gt;     person("Thomas Lin", "Pedersen", , "thomas.pedersen@rstudio.com", 
#&gt;            role = c("aut", "cre"), comment = c(ORCID = "0000-0002-5147-4711")),
#&gt;     person("Kohske", "Takahashi", role = "aut"),
#&gt;     person("Claus", "Wilke", role = "aut",
#&gt;            comment = c(ORCID = "0000-0002-7470-9261")),
#&gt;     person("Kara", "Woo", role = "aut",
#&gt;            comment = c(ORCID = "0000-0002-5125-4188")),
#&gt;     person("Hiroaki", "Yutani", role = "aut",
#&gt;            comment = c(ORCID = "0000-0002-3385-7233")),
#&gt;     person("Dewey", "Dunnington", role = "aut",
#&gt;            comment = c(ORCID = "0000-0002-9415-4582")),
#&gt;     person("RStudio", role = c("cph", "fnd"))
#&gt;     )
#&gt; 
#&gt; </span><span style='color: #555555;'>Enhances: </span><span>sp
#&gt; 
#&gt; </span><span style='color: #555555;'>Imports: </span><span>
#&gt; digest, glue, grDevices, grid, gtable (&gt;= 0.1.1), isoband,
#&gt; MASS, mgcv, rlang (&gt;= 0.3.0), scales (&gt;= 0.5.0), stats, tibble,
#&gt; withr (&gt;= 2.0.0)
#&gt; 
#&gt; </span><span style='color: #555555;'>Depends: </span><span>R (&gt;= 3.2)
#&gt; 
#&gt; </span><span style='color: #555555;'>Suggests: </span><span>
#&gt; covr, dplyr, ggplot2movies, hexbin, Hmisc, knitr, lattice,
#&gt; mapproj, maps, maptools, multcomp, munsell, nlme, profvis,
#&gt; quantreg, RColorBrewer, rgeos, rmarkdown, rpart, sf (&gt;= 0.7-3),
#&gt; svglite (&gt;= 1.2.0.9001), testthat (&gt;= 2.1.0), vdiffr (&gt;= 0.3.0)
#&gt; 
#&gt; </span><span style='color: #555555;'>Reverse depends: </span><span>
#&gt; ACSNMineR, AFheritability, afmToolkit, alakazam, alookr, AmpliconDuo, Anaquin, AneuFinder, apisensr, applicable, bacon, BasketballAnalyzeR, bayesDP, BCellMA, bde, BDMMAcorrect, bhm, biclustermd, bmscstan, bootnet, bpcp, braidReports, btergm, bunching, caret, CellNOptR, centralplot, ceterisParibus, changepoint.geo, ChaosGame, CHETAH, ChIPQC, chromstaR, cjoint, classifierplots, climwin, clustEff, clustrd, CNVrd2, CNVScope, coefplot, cogena, cohorttools, corkscrew, CoSMoS, CountClust, CrispRVariants, crmPack, Crossover, CRTgeeDR, cummeRbund, curatedBreastData, cystiSim, cytofan, dae, DaMiRseq, dampack, dartR, dbmss, decompTumor2Sig, DeconRNASeq, Deducer, deltaGseg, DendroSync, DengueRT, DepthProc, DEqMS, dfexplore, DHBins, diathor, diffEnrich, diffeR, dittoSeq, DoTC, dotwhisker, dslice, dtwSat, dvqcc, dynr, Eagle, earlywarnings, ecap, eeptools, egg, EnhancedVolcano, EnsCat, entropart, EpiCurve, epihet, episensr, erccdashboard, EventStudy, FactoClass, factoextra, Factoshiny, fbroc, FisherEM, fishmove, flippant, forestmodel, FourCSeq, freqparcoord, frequency, func2vis, funcharts, FunciSNP, funLBM, gapfill, garma, GARS, gcerisk, gde, GENEAsphere, genlogis, genomeplot, GenomicOZone, geomnet, geotoolsR, gespeR, ggallin, ggalluvial, GGally, ggalt, gganimate, ggbeeswarm, ggbio, ggbuildr, ggcharts, ggcorrplot, ggcyto, ggdag, ggDCA, ggdemetra, ggedit, ggetho, ggfocus, ggforce, ggformula, ggfortify, ggghost, gggibbous, ggh4x, gghalves, gghighlight, ggimage, gginnards, ggip, gglorenz, ggmap, ggmcmc, ggmosaic, ggmulti, ggnetwork, ggOceanMaps, ggparallel, ggparty, ggpmisc, ggpol, ggpolypath, ggpubr, ggraph, ggraptR, ggrepel, ggROC, ggseas, ggside, ggsn, ggspectra, ggswissmaps, ggtern, ggTimeSeries, ggvenn, ggvoronoi, ggwordcloud, glow, GOplot, GOxploreR, gpmap, granovaGG, GRENITS, GSCA, gsDesign, gstar, Guitar, GWAS.BAYES, gwdegree, h5vc, HCmodelSets, HiLDA, Hmisc, HRM, hyperSpec, iarm, iBMQ, iCARH, iCellR, ICSsmoothing, ICtest, idm, ifaTools, IMAS, infinitefactor, inlabru, InSilicoVA, interplot, inti, invacost, ioanalysis, IPDfromKM, IsoformSwitchAnalyzeR, itan, itemanalysis, iteremoval, itraxR, IVAS, kayadata, LambertW, lavaSearch2, lcc, learnstats, likeLTD, likert, LinkHD, listdtr, llbayesireg, logihist, longRPart2, loon.ggplot, lsbclust, MAFDash, malariaAtlas, maser, MCAvariants, mccf1, MCMC.OTU, MCMC.qpcr, mcprofile, metacart, metaforest, meteogRam, methimpute, methylumi, mgcViz, mhtboot, microbiome, MicroNiche, MIMOSA, MinEDfind, MineICA, missRows, MIXFIM, mixKernel, mixOmics, MixOptim, MLSeq, mlxR, mnreadR, monocle, MortalityTables, motoRneuron, mrfDepth, multilevelPSA, MultisiteMediation, NanoMethViz, NBLDA, Nebulosa, NetworkChange, ngsReports, NIRStat, nonmem2R, npde, oceanwaves, OptimalRerandExpDesigns, optimos.prime, orgR, OriGen, overlapping, pafr, pagoo, PairedData, pAnalysis, PASenseWear, PAsso, PASWR2, pathVar, pauwels2014, PCAtools, PedCNV, pequod, perry, personalized, perspectev, phase1PRMD, phase1RMD, phenModel, phenoTest, photosynthesis, phyloseqGraphTest, Pi, pitchRx, pkggraph, PKreport, plantecowrap, PlasmaMutationDetector, plothelper, plotly, plotROC, pointRes, PortfolioEffectHFT, PPInfer, PPtreeViz, precintcon, primer, PRISMA, ProbBayes, profileR, PupillometryR, pvldcurve, qqplotr, qrqc, quadrupen, QualInt, quickpsy, rabhit, radiant.data, RAM, randomizeR, RATest, RatingScaleReduction, RcmdrPlugin.KMggplot2, rdacca.hp, RDAVIDWebService, recoup, reinsureR, Repliscope, rise, rms, RNAseqNet, RNASeqR, RnBeads, Rnits, robCompositions, robmed, robustHD, rorutadis, RpeakChrom, Rqc, rrecsys, RSA, rSEA, RsSimulx, rstan, rtip, rwty, samExploreR, savR, scAlign, scater, scatterpie, schex, scoper, SCOUTer, scPipe, scRepertoire, SDLfilter, sdStaf, season, SEERaBomb, selfea, sensiPhy, seqsetvis, SetMethods, sglr, shazam, shinySIR, simmer.plot, simmr, SmarterPoland, SMFI5, soc.ca, SparseBiplots, SparseFactorAnalysis, sparsereg, spatialClust, SPEI, spqn, sprm, ssviz, staRdom, StatMatch, statVisual, sugrrants, SummarizedBenchmark, survHE, survminer, survRatio, swa, SWMPr, synlet, TarSeqQC, TcGSA, tdr, texmex, texteffect, tigger, timeline, TissueEnrich, TPD, TreeDep, TriMatch, TripleR, tspmeta, twangRDC, useful, ushr, varian, variancePartition, vdg, visae, VplotR, vrcp, waffle, weightr, WRTDStidal, xkcd, Xplortext, xpose, YAPSA, zooaRch
#&gt; 
#&gt; </span><span style='color: #555555;'>Reverse suggests: </span><span>
#&gt; abctools, abd, activatr, activityCounts, AdaptGauss, adept, adklakedata, adoptr, afex, aggregateBioVar, aghq, agridat, ahpsurvey, aire.zmvm, airGR, airt, ALA4R, alfred, AlleleShift, AmpGram, AMR, amt, antaresViz, antitrust, aopdata, apsimx, ar.matrix, archetypes, archivist, arcos, arctools, ARPobservation, ARTool, ashr, asymptor, AugmenterR, automap, automultinomial, autothresholdr, av, avocado, babette, bacondecomp, bambu, bang, baRulho, BAS, base2grob, BaseSet, BatchGetSymbols, bayesbio, bayestestR, bbmle, bcdata, BCEA, bcmaps, bcp, beautier, belg, bench, benchmarkfdrData2019, benchmarkme, benchr, benthos, bestNormalize, BetaBit, bfw, bib2df, BiDimRegression, binom, binomialMix, binsegRcpp, bioassayR, biobroom, biogas, biogram, Bioi, bioset, Biostatistics, BioTIP, BIS, bisect, bitmexr, BlandAltmanLeh, blastula, blockCV, blockRAR, BMRSr, bodenmiller, BOJ, bomrang, bootUR, borrowr, brainGraph, breathteststan, BRGenomics, bridgedist, briskaR, brokenstick, broom, broom.helpers, broom.mixed, brotli, brunnermunzel, BSDA, bsem, bssm, bsub, bulletcp, bwsTools, cancensus, CancerGram, cansim, caRecall, CarletonStats, cate, CDatanet, CellaRepertorium, CellBench, cellHTS2, celltrackR, Census2016, CGGP, cheatR, checkmate, cherryblossom, childsds, chilemapas, ChillModels, choroplethrMaps, chorrrds, chron, ciftiTools, ciTools, civis, ClassifyR, CLLmethylation, clusternomics, ClusterSignificance, cmocean, codingMatrices, codyn, collapse, colorednoise, colorspace, comparer, COMPASS, compboost, comtradr, consensusOV, consensusSeekeR, constellation, contactdata, contextual, contoureR, ContourFunctions, contrast, CoRegFlux, CornerstoneR, correlation, countytimezones, covafillr, CPAT, cpp11, cpsvote, crandep, crawl, CropDetectR, crosstable, crosstalk, cSEM, CTD, curatedAdipoChIP, curatedAdipoRNA, curatedMetagenomicData, cusum, CVEK, cvequality, cyclomort, Cyclops, czechrates, DAISIE, dams, danstat, dasper, dataresqc, dataRetrieval, datarobot, datasauRus, datplot, daymetr, DeclareDesign, deconvolveR, DeepBlueR, deGradInfer, densvis, denvax, depmap, derfinder, derivmkts, descstat, designr, detrendr, dexter, dexterMST, DGCA, DGVM3D, dielectric, diffuStats, DImodels, dimRed, dint, DirectEffects, directlabels, disclapmix, discrim, diseq, DisImpact, disk.frame, distributions3, dobin, dodgr, dominanceanalysis, dpseg, drake, drawer, DrBats, dscore, DSjobtracker, dstack, dynutils, earthtones, EbayesThresh, ec50estimator, Ecfun, echo.find, echor, ecoCopula, ecotox, edfReader, edgarWebR, edge, eemR, eesim, EFDR, effectsize, EGAnet, EHR, eia, embed, emmeans, EMMIXmfa, emuR, enrichwith, eph, epigraphdb, epiR, EpiTxDb, eq5d, equatiomatic, escalation, esetVis, etl, etm, eurlex, europepmc, eurostat, evaluate, exact2x2, expss, ExtDist, eyelinker, fable.prophet, facerec, factory, fastRG, FDboost, fddm, fec16, fedregs, feisr, fermicatsR, FField, fgeo.analyze, fhidata, fitbitScraper, fitdistrplus, fitHeavyTail, fitzRoy, fivethirtyeight, flacco, flexrsurv, flexsurv, flextable, flowTime, FMM, forcats, forrel, forwards, foster, fourierin, fractional, fredr, frequencyConnectedness, FRESA.CAD, frontiles, frost, FSelectorRcpp, functiondepends, funData, funrar, funtimes, furrowSeg, fuser, FusionLearn, fuzzyjoin, gaiah, ganalytics, GAPGOM, gapminder, gcatest, GDAdata, gear, gender, geneplast, GENESIS, GenoGAM, GenomicDataCommons, genphen, geobr, geodiv, geofi, geoknife, GetBCBData, getCRUCLdata, GetDFPData, GetDFPData2, GetLattesData, GetQuandlData, GetTDData, ggeffects, ggROC, ghcm, ghibli, ghypernet, gifski, GillespieSSA2, git2rdata, glm.predict, GLMcat, glmGamPoi, glmmTMB, glmpca, glue, googlesheets, gplite, grainchanger, graphhopper, graphlayouts, grates, grattan, Greg, gridDebug, gridExtra, GROAN, groupdata2, groupedstats, growthcurver, GSgalgoR, gsloid, GSODR, GSVA, gtable, guardianapi, gutenbergr, h2o, hal9001, Haplin, harrietr, hazus, hbbr, HDCytoData, hdfqlr, HDiR, heddlr, hereR, hermiter, heuristica, HIBAG, highcharter, hilbertSimilarity, hIRT, HistData, historydata, hmclearn, HMP16SData, HMP2Data, holodeck, horseshoe, hpa, hsrecombi, HTLR, httk, hues, hutils, huxtable, hydroscoper, hystReet, i2dash, ialiquor, icarus, ICON, iconr, icr, icrf, idiogramFISH, IgGeneUsage, igoR, IHW, imagefluency, imager, imgrec, imputeFin, installr, InterVA5, iNZightPlots, ipmisc, ipumsr, irg, isoband, isocat, ivmte, JacobiEigen, jmvcore, JointAI, kamila, keras, kergp, kfigr, kibior, kiwisR, klsh, knitrBootstrap, KSD, LAGOSNE, Lahman, latex2exp, lda, LDheatmap, ldsr, LEAP, lfa, lgrdata, lillies, lingtypology, listdown, lme4, LocalControl, localIV, logitnorm, logitr, lognorm, longCatEDA, loo, LOPART, lotri, LPWC, LSMRealOptions, lspline, ltbayes, lutz, MAc, macleish, MACPET, MAd, maddison, madrat, magick, MANOVA.RM, mapchina, mapi, mapsapi, mapsFinland, margins, marked, marketr, marqLevAlg, MARSS, matchingMarkets, matrixStats, matrixStrucTest, matsindf, MBNMAdose, MBNMAtime, mcbette, medicalrisk, medicare, memor, metagenomeFeatures, MetaStan, methcon5, methylCC, metricsgraphics, mets, mfGARCH, mgc, MGDrivE2, MGLM, microbenchmark, mifa, miic, mikropml, milr, mindicador, missMethods, mistat, mistr, mixdir, mixIndependR, MixMatrix, mleap, mlogit, mlr3pipelines, mlr3proba, mlrMBO, MLVSBM, mmb, mmpf, MNLpred, modcmfitr, modelbased, modelr, modelsummary, MODISTools, MODIStsp, ModStatR, MOEADr, moexer, momentuHMM, MoMPCA, mongolite, MonteCarlo, moonBook, mortAAR, mosaic.find, mosaicData, mosmafs, MOSS, move, mrbayes, MrSGUIDE, MSG, mstate, mthapower, MTLR, MTPS, MultiBD, multiROC, munsell, musica, mvSLOUCH, mwaved, mwshiny, nanny, nasapower, naspaclust, nationwider, NBAloveR, NBAMSeq, nbapalettes, nbTransmission, nc, NestLink, NetMix, netrankr, netSmooth, networkABC, neurobase, NewWave, nhdplusTools, NHSRdatasets, NitrogenUptake2016, nlraa, nls.multstart, NlsyLinks, nLTT, NNTbiomarker, nntrf, noaaoceans, nomisr, nosoi, NPARC, nprcgenekeepr, npsurvSS, nseval, nzelect, observer, OECD, officedown, officer, ohenery, OmnipathR, opendatatoronto, OptCirClust, optimLanduse, optimParallel, optiSel, OPWeight, ordinalClust, Organism.dplyr, outcomerate, OVESEG, oxcAAR, oxcovid19, packcircles, padma, padr, pafdR, pagenum, pagoda2, paletteer, palettetown, palmerpenguins, pals, pander, parallelDist, parameters, ParamHelpers, paramtest, parcoords, parlitools, parSim, passport, passt, PBImisc, pbixr, pcalg, pcaMethods, pcFactorStan, pct, pder, pdi, PDQutils, pdSpecEst, pdxTrees, PeakError, PeakSegDisk, PeakSegDP, PeakSegJoint, PeakSegOptimal, peptider, perccalc, performance, PerformanceAnalytics, permutes, PheCAP, PhosR, photobiologyFilters, photobiologyInOut, photobiologyLamps, photobiologyLEDs, photobiologyPlants, photobiologySensors, photobiologySun, phyr, physiology, pivottabler, pkgnet, PKNCA, plotscale, plyranges, PNWColors, pocketapi, pointblank, pollen, pollster, pollstR, Polychrome, polymapR, polyRAD, pomp, popEpi, popReconstruct, portalr, povcalnetR, powerlmm, PPforest, pqsfinder, PRDA, prepplot, prevtoinc, proActiv, probably, pROC, profr, profvis, ProjectTemplate, projmgr, PRROC, PSCBS, psd, pseval, pspline.inference, psychonetrics, psymonitor, pwr, pysd2r, qcmetrics, QFeatures, QSutils, qualmap, qualvar, quantdr, QuantNorm, questionr, queuecomputer, R.SamBada, R2admb, r2rtf, r3PG, R3port, r5r, R6, R6P, radarBoxplot, ragtop, raincpc, RandomForestsGLS, rangeMapper, Raquifer, rasterVis, raw, rayshader, rbefdata, rbi, RcmdrPlugin.MA, rco, RcppHungarian, RCzechia, Rdca, rdefra, rdhs, rdi, RDML, rdtLite, Rdtq, readabs, readrba, readsdr, rearrr, recipes, Recon, recountmethylation, REddyProc, regionReport, rem, remap, reporter, representr, retrodesign, reval, revdbayes, revulyticsR, rex, rfordummies, RforProteomics, RGCCA, RGENERATEPREC, rhdf5, rhosa, ricu, Rilostat, riskParityPortfolio, rivr, rKolada, rlc, rLFT, Rmbal, rmcorr, rmdfiltr, rmdformats, rmdpartials, rnbp, rnpn, RoBMA, robustbase, rODE, roloc, rpact, rpanel, rpdo, rpf, rPref, Rpvt, RQEntangle, rrd, Rrelperm, RRphylo, rrscale, rsample, rsetse, rsinaica, rsppfp, rstpm2, rsvd, rsvg, rsyncrosim, rtematres, rtf, Rthingsboard, rtide, RTransferEntropy, rTRM, rtweet, rubias, runstats, rvinecopulib, RWmisc, sadists, safetyGraphics, SAGMM, salem, salesforcer, SAMBA, SamplerCompare, SAMtool, sand, sarks, sbo, SC2API, scales, SCATE, scdhlm, scico, scmeth, scone, scoringTools, scp, scPCA, scry, SDaA, SDEFSR, sdmpredictors, sdmvspecies, seer, seewave, selectKSigs, seqbias, seqhandbook, SEtools, sf, sfcr, sfnetworks, sftrack, sgmcmc, shades, shapr, shiny, shinybrms, shinydlplot, shinyjqui, shinyMobile, shinyPredict, shinyWidgets, SightabilityModel, sigmoid, signnet, signs, SimDesign, simglm, simhelpers, simITS, simplifyEnrichment, simstandard, simstudy, SimSurvey, simTool, simulator, Single.mTEC.Transcriptomes, SingleCaseES, SingleR, sitmo, SiZer, sjmisc, sjstats, slim, slinky, smacof, smcfcs, SMFilter, snifter, socialmixr, sociome, socviz, Sofi, soilDB, SomaticCancerAlterations, sotkanet, SourceSet, sparklyr, sparseMVN, spatialsample, spbabel, spdep, SPONGE, spongebob, spread, srvyr, SSOSVM, ssPATHS, stabm, starm, stars, states, stats19, statsExpressions, stlcsb, StratifiedSampling, streamDepletr, strider, strm, struct, StructFDR, StructuralVariantAnnotation, sugarbag, SunsVoc, SuperLearner, swag, swfdr, swmmr, T4transport, tableone, taipan, TAPseq, taRifx, tbl2xts, tboot, tbrf, tcensReg, TeachingDemos, tealeaves, testassay, testDriveR, texreg, textplot, TFEA.ChIP, tfestimators, TFutils, themis, threesixtygiving, tibble, tictactoe, tidyBF, tidybulk, tidycat, tidycensus, tidyfst, tidygeocoder, tidyhydat, tidyjson, tidylo, tidync, tidytext, tidytransit, tidyxl, tigris, tikzDevice, TimeProjection, tint, tlsh, tmap, Tmisc, tongfen, totalcensus, tourr, tracerer, trackdf, TrafficBDE, TrajDataMining, tramME, transx, trapezoid, treasuryTR, treecm, treedater, treeio, treespace, treestructure, TreeSummarizedExperiment, trending, tRNAscanImport, truelies, ts.extend, tsbox, tsfeatures, tsibble, tsibbledata, tsibbletalk, TSRchitect, TSrepr, tufterhandout, tvgeom, UBL, ukgasapi, ukpolice, unvotes, usdata, UsingR, usmap, uSORT, utilities, vacuum, vagalumeR, VancouvR, VariantAnnotation, vcdExtra, vcfR, vdar, vdiffr, velociraptor, venn, vimp, vinereg, vioplot, vipor, viridisLite, visreg, vlad, volleystat, vote, vroom, vtree, warbleR, WaveSampling, waydown, wbstats, wdpar, weathercan, WeightedROC, wesanderson, whereami, widyr, wildlifeDI, WoodSimulatR, wordbankr, WordR, WRS2, wrswoR, wrswoR.benchmark, xaringanthemer, xgb2sql, xgboost, XICOR, XLConnect, yardstick, zenplots, zinbwave, zoo, zoolog, ztpln, ZVCV
#&gt; 
#&gt; </span><span style='color: #555555;'>Reverse enhances: </span><span>flowr, LOLA, mcaGUI, surveillance, tis
#&gt; 
#&gt; </span><span style='color: #BB00BB;font-weight: bold;'>
#&gt; Tip:</span><span style='color: #BB00BB;'> Use </span><span style='color: #BB00BB;font-weight: bold;'>go("ggplot2","manual")</span><span style='color: #BB00BB;'> to view the manual of package 'ggplot2' and </span><span style='color: #BB00BB;font-weight: bold;'>go("ggplot2","website")</span><span style='color: #BB00BB;'> to visit its website (if any is provided).</span><span></div><div class='input'>
</div></span></pre>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top">
      <h2 data-toc-skip>Contents</h2>
    </nav>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Joachim Zuckarelli.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


